tracer/src/Datadog.Trace/Iast/Aspects/System.IO/DirectoryInfoAspect.cs

```diff
@@ -0,0 +1,64 @@
+// <copyright file="DirectoryInfoAspect.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using Datadog.Trace.Iast.Dataflow;
+
+namespace Datadog.Trace.Iast.Aspects;
+
+/// <summary> DirectoryInfoAspect class aspects </summary>
+[AspectClass("mscorlib,System.IO.FileSystem,System.Runtime", AspectType.Sink, VulnerabilityType.PathTraversal)]
+[global::System.ComponentModel.Browsable(false)]
+[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+public partial class DirectoryInfoAspect
```

Comment:
nitpick, but as it's already a partial class, maybe we could have one DirectoryInfoAspect.Framework.cs and one for Core, that would avoid all the #id #endif

Reply:
I personally like to have in the same place a list of all the covered methods for all the frameworks, but it is just a personal preference. Another option would be to group all the .net framework methods in the same #if section and avoid having many #if sections. I like to have the methods ordered by name to spot easily all the overloads at once, but, again, it is just a personal preference.

Reply:
In fact, I'd remove the `partial` modifier

Reply:
Agree. I have removed the partial word from the aspects.